packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -442,7 +443,11 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
       metadata: { Modality: this.modality },
       scaling: this.scaling,
       hasPixelSpacing: this.hasPixelSpacing,
-      calibration: this.calibration,
+      calibration: Object.assign(
```

Comment:
Can you use the ... notation for consistency - or change the other examples to use Object.assign, whichever you prefer.